lib/modules/newCommentCount.js

```diff
@@ -266,20 +272,19 @@ function stopTracking(id) {
 }
 
 async function checkSubscriptions() {
+	const now = Date.now();
+	lastSubscriptionCheckStorage.set(now);
+
 	for (const [id, subscription] of Object.entries(await entryStorage.getAll())) {
 		const { subscriptionDate } = subscription;
 		if (subscriptionDate) {
-			const lastCheck = parseInt(subscription.lastCheck, 10) || 0;
 			// If it's been subscriptionLength days since we've subscribed, we're going to delete this subscription...
-			const now = Date.now();
 			if ((now - subscriptionDate) > (DAY * parseInt(module.options.subscriptionLength.value, 10))) {
 				unsubscribe(id);
 			}
-			// if we haven't checked this subscription in 5 minutes, try it again...
-			if ((now - lastCheck) > 300000) {
-				entryStorage.patch(id, { lastCheck: now });
-				checkThread(id, subscription);
-			}
+
+			entryStorage.patch(id, { lastCheck: now });
```

Comment:
I don't think `lastCheck` is read anymore, so if that's the case might as well remove it from here and the type on line 89